plugins/tpm-eventlog/fu-tpm-eventlog.c

```diff
@@ -50,19 +51,29 @@ fu_tmp_eventlog_process (const gchar *fn, gint pcr, GError **error)
 
 	for (guint i = 0; i < items->len; i++) {
 		FuTpmEventlogItem *item = g_ptr_array_index (items, i);
+		if (item->pcr > max_pcr)
+			max_pcr = item->pcr;
 		if (pcr >= 0 && item->pcr != pcr)
 			continue;
 		fu_tpm_eventlog_item_to_string (item, 0, str);
 		g_string_append (str, "\n");
 	}
+	if (pcr > max_pcr) {
+		g_set_error (error, G_IO_ERROR, G_IO_ERROR_INVALID_DATA, "invalid PCR %d", pcr);
```

Comment:
line is a bit long

Reply:
Also, PCR should be %x, no?

Reply:
Is it? I guess right now we're only going up to PCR 9 on my testing system, but I've never seen a PCR above referred to as 0xa.

Reply:
ohh true! ignore me, sorry.